data_summaries/AgentInstruct.json

```diff
@@ -0,0 +1,301 @@
+        ],
+        "License Notes": "The authors collected data on various products from Amazon.com and retrieved text and crowdsourced instructions from annotators. It is possible that some of these annotations have additional licenses. Therefore, the license pertains to the framework that was used to create the dataset and is mentioned by the authors.",
+        "License Verified By": "Christopher",
+        "Dataset Filter IDs": [
+            "webshop"
+        ]
+    },
+    "AgentInstruct-knowledgegraph": {
+        "Unique Dataset Identifier": "AgentInstruct-knowledgegraph",
+        "Dataset Name": "knowledgegraph",
+        "Paper Title": "AgentBench: Evaluating LLMs as Agents",
+        "Dataset URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/kg",
+        "GitHub URL": "https://github.com/THUDM/AgentBench",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/kg",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2310.12823",
+        "Semantic Scholar Corpus ID": 260682249,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Chain-of-Thought",
+            "Instruction Following"
+        ],
+        "Text Sources": [
+            "freebase"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "GrailQA",
+            "ComplexWebQuestions",
+            "GraphQuestions"
+        ],
+        "Creators": [
+            "Tsinghua University",
+            "The Ohio State University",
+            "UC Berkeley"
+        ],
+        "Licenses": [
+            {
+                "License": "Unspecified",
+                "License URL": ""
+            }
+        ],
+        "License Notes": "The paper cites an anonymous source from 2023 that is currently under review. It is unclear what that means for the license. The original Freebase dataset, which seems to be a part of the source, is licensed under the Creative Commmon Attribution License.",
+        "License Verified By": "Christopher",
+        "Dataset Filter IDs": [
+            "kg"
+        ]
+    },
+    "AgentInstruct-operatingsystem": {
+        "Unique Dataset Identifier": "AgentInstruct-operatingsystem",
+        "Dataset Name": "operatingsystem",
+        "Paper Title": "AgentBench: Evaluating LLMs as Agents",
+        "Dataset URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/os",
+        "GitHub URL": "https://github.com/THUDM/AgentBench",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/os",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2310.12823",
```

Comment:
Should we put this paper if this is the source one? https://arxiv.org/pdf/2308.03688.pdf